File: client/iih_stub/iih.ci_stub/iih.ci.mr/nu/obstetrics/childbirthrecord/d/EnumExist.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace iih.ci.mr.nu.obstetrics.childbirthrecord.d
{
   public enum EnumExist {
	/// <summary>
	/// 有
	/// </summary>
    [Description("有")]
    EXIST=0,
	/// <summary>
	/// 无
	/// </summary>
    [Description("无")]
    NOEXIST=1,
   }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/nu/obstetrics/childbirthrecord/d/EnumExist.java
package iih.ci.mr.nu.obstetrics.childbirthrecord.d;

import xap.sys.appfw.orm.desc.ent.dataobject.DmEnumDesc;

public interface EnumExist {

    @DmEnumDesc(name="有",description="有伤裂")
	public static final Integer EXIST=0; //有
    @DmEnumDesc(name="无",description="无伤裂")
	public static final Integer NOEXIST=1; //无
}	
